packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -207,6 +207,23 @@ export class Renderer extends EventDispatcher {
     return this[$arRenderer] != null && this[$arRenderer].isPresenting;
   }
 
+  preRender(scene: ModelScene, t: number, delta: number) {
```

Comment:
What is the intended purpose of this new public API? (please add docs)

Reply:
Done.